less/index.less

```diff
@@ -50,10 +50,379 @@ table {
+			justify-content: space-around;
+
+			@media (max-width: 500px) {
+				flex-wrap: wrap;
+			}
+
+			& li {
+				width: 100px;
+				height: 100px;
+				border-radius: 50%;
+				text-align: center;
+				line-height: 100px;
+				vertical-align: middle;
+				background: @secondaryPurple;
+				margin: 20px 0;
+
+				@media (max-width: 500px) {
+						margin: 20px;
+					}
+
+				& i {
+					padding: 20px;
+				}
+			}
+		}
+
+	}
+
+	.box-four {
+		display: flex;
+		align-self: center;
+		flex-direction: column;
+		background: @primaryPurple;
+		width: 800px;
+		border-radius: 5px;
+		margin-top: 20px;
+		color: @defaultWhite;
+
+		@media (max-width: 500px) {
+			width: 400px;
+		}
+
+		& h1 {
+			.BoxHeading;
+			margin: 20px 40px 20px;
+		}
+
+		& ul li {
+			font-size: 1.6rem;
+			margin: 0 40px 20px;
+
+			& span {
+				font-style: italic;
+				padding: 0 10px;
+			}
+		}
+	}
+
+	.box-five {
+		display: flex;
+		align-self: center;
+		flex-direction: column;
+		background: @primaryPurple;
+		width: 800px;
+		border-radius: 5px;
+		margin-top: 20px;
+		color: @defaultWhite;
+
+		@media (max-width: 500px) {
+			width: 400px;
+		}
+
+		& h1 {
+			.BoxHeading;
+			.BoxSpacing;
+		}
+
+		& ul li {
+			font-size: 1.6rem;
+			// TRBL
+			margin: 0 40px 20px;
+
+			& span {
+				font-style: italic;
+				padding: 0 10px;
+			}
+		}
+	}
+
+	.box-contact {
+		display: flex;
+		align-self: center;
+		flex-direction: column;
+		background: @primaryPurple;
+		width: 800px;
+		border-radius: 5px;
+		margin-top: 20px;
+		color: @defaultWhite;
+		margin-bottom: 100px;
+
+		@media (max-width: 500px) {
+			width: 400px;
+			margin-bottom: 50px;
+		}
+
+		& h1 {
+			.BoxHeading;
+			.BoxSpacing;
+		}
+
+		& form {
+			display: flex;
+			justify-content: center;
+			flex-direction: column;
+			align-items: center;
+
+			& input {
+				background: transparent;
+				border: 1px solid @secondaryPurple;
+				padding: 20px;
+				border-radius: 5px;
+				width: 500px;
+				margin: 10px 40px 0px;
+				color: white;
+				font-size: 1.6rem;
+
+				&::-webkit-input-placeholder {
+					color: #111e33;
+				}
+
+				@media (max-width: 500px) {
+					width: 315px;
+				}
+			}
+
+			& textarea {
+				background: transparent;
+				border: 1px solid @secondaryPurple;
+				padding: 20px;
+				border-radius: 5px;
+				width: 500px;
+				height: 300px;
+				margin: 10px 40px 0;
+				color: white;
+				resize: none;
+				font-size: 1.6rem;
+
+				&::-webkit-input-placeholder {
+					color: #111e33;
+				}
+
+				@media (max-width: 500px) {
+					width: 315px;
+				}
+			}
+
+			& button {
+				background: transparent;
+				border: 1px solid @secondaryPurple;
+				padding: 20px;
+				margin: 10px 0 30px;
+				width: 500px;
+				font-size: 1.6rem;
+				color: @secondaryPurple;
+				transition: 0.3s;
+				outline: none;
+
+				&:hover {
+					color: #fff;
+					background: @secondaryPurple;
+				}
+
+
+				@media (max-width: 500px) {
+					width: 315px;
+				}
+			}
+		}
```

Comment:
Great job as usual, Skyelar!! I love the icons. Great layout. Overall, excellent job. I can't wait to see what you're going to come up with next!!